agents/mysql/slowlog/slowlog_test.go

```diff
@@ -41,6 +42,46 @@ func getDataFromFile(t *testing.T, filePath string, data interface{}) {
 	require.NoError(t, err)
 }
 
+func TestSlowLogMakeBucketsInvalidUTF8(t *testing.T) {
+	const agentID = "/agent_id/73ee2f92-d5aa-45f0-8b09-6d3df605fd44"
+	periodStart := time.Unix(1557137220, 0)
+
+	parsingResult := event.Result{
+		Class: map[string]*event.Class{
+			"example": {
+				Metrics:     &event.Metrics{},
+				Fingerprint: "set lock_wait_timeout=?\xff",
```

Comment:
Can we make a somewhat more real test? That show what data we lose from the real requests.